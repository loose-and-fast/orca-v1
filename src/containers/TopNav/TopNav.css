@import url("https://fonts.googleapis.com/css?family=Orbitron");

.TopNav {
  height: 100vh;
  background-image: url("../../assets/open.jpg");
  background-size: cover;
  font-family: "Orbitron", sans-serif;
}

.TopNav .bullet {
  font-size: 2rem;
  font-weight: bold;
  color: white;
}
