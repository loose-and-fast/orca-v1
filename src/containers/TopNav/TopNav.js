import React, { Component } from "react";
// import PropTypes from "prop-types";
import "./TopNav.css";
import NavItem from "components/NavItem/NavItem";
import { Row } from "react-bootstrap";

class TopNav extends Component {
  state = {};

  render() {
    return (
      <div className="TopNav">
        <Row>
          {/* <img src={image} /> */}
          <NavItem className="item" text="WORK" />
          <span className="bullet">&middot;</span>
          <NavItem className="item" text="ABOUT" />
          <span className="bullet">&middot;</span>
          <NavItem className="item" text="CONTACT" />
        </Row>
      </div>
    );
  }
}

TopNav.propTypes = {};

TopNav.defaultProps = {};

export default TopNav;
