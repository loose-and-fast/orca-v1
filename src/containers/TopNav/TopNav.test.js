import React from 'react';
import { shallow } from 'enzyme';
import TopNav from './TopNav';

test('TopNav renders', () => {
  const wrapper = shallow(<TopNav />);
  expect(wrapper).toMatchSnapshot();
});
