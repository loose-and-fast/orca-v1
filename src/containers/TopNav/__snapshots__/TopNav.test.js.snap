// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNav renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TopNav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Row
        bsClass="row"
        componentClass="div"
>
        <NavItem
                className="item"
                text="WORK"
        />
        <span
                className="bullet"
        >
                ·
        </span>
        <NavItem
                className="item"
                text="ABOUT"
        />
        <span
                className="bullet"
        >
                ·
        </span>
        <NavItem
                className="item"
                text="CONTACT"
        />
</Row>,
      "className": "TopNav",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bsClass": "row",
        "children": Array [
          <NavItem
            className="item"
            text="WORK"
/>,
          <span
            className="bullet"
>
            ·
</span>,
          <NavItem
            className="item"
            text="ABOUT"
/>,
          <span
            className="bullet"
>
            ·
</span>,
          <NavItem
            className="item"
            text="CONTACT"
/>,
        ],
        "componentClass": "div",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "className": "item",
            "text": "WORK",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "·",
            "className": "bullet",
          },
          "ref": null,
          "rendered": "·",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "className": "item",
            "text": "ABOUT",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "·",
            "className": "bullet",
          },
          "ref": null,
          "rendered": "·",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "className": "item",
            "text": "CONTACT",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Row
          bsClass="row"
          componentClass="div"
>
          <NavItem
                    className="item"
                    text="WORK"
          />
          <span
                    className="bullet"
          >
                    ·
          </span>
          <NavItem
                    className="item"
                    text="ABOUT"
          />
          <span
                    className="bullet"
          >
                    ·
          </span>
          <NavItem
                    className="item"
                    text="CONTACT"
          />
</Row>,
        "className": "TopNav",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "row",
          "children": Array [
            <NavItem
              className="item"
              text="WORK"
/>,
            <span
              className="bullet"
>
              ·
</span>,
            <NavItem
              className="item"
              text="ABOUT"
/>,
            <span
              className="bullet"
>
              ·
</span>,
            <NavItem
              className="item"
              text="CONTACT"
/>,
          ],
          "componentClass": "div",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "className": "item",
              "text": "WORK",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "·",
              "className": "bullet",
            },
            "ref": null,
            "rendered": "·",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "className": "item",
              "text": "ABOUT",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "·",
              "className": "bullet",
            },
            "ref": null,
            "rendered": "·",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "className": "item",
              "text": "CONTACT",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
